# Babys_First_AMF_Phylogeny
Creating a phylogeny from fungi ASV fragments

#first commit f096e4b9107ee656fff7da5c36b1fbf757c391e6
#first edit for v1 on 0110 6 Nov 2022
#edit.id=js781
#propri; private

#list to write
Project title-review
participants-review
reference to our institution and this course-review
brief: research rationale, objectives, outcomes and impact-review
data sources
bioinformatics tools - with citations

Project Title: What would it look like visualizing and comparing Klamath Falls, Oregon's Arbuscular Mycorrhizae Fungi (AMF) organisms with Indonesia's AMF?

Secret tiktok login packet sniffer
#look at this when tired

Participants: js781, cc349

Heading: Institution and Course
Body: California State Polytechnic University, Humboldt; BIOL 580 (43469)

Heading: Research Vision, Milestones to Reach, and Expected Outcomes and Impacts.
Body: Js781 wishes to taxonomically understand the relationship between Banyuasin and Klamath Falls (KF) sites' AMF composition. Banyuasin (B) is a coastal
region in South Sumatra, Indonesia. The area has tidally influenced marsh villages that see anthropological activity. Due to salt and flooding proximity,
B's AMF taxon data can provide a contrast to KF sites. While salt gradient and zonation with tidal influence may affect niche partitioning, they may be
key observations to contrast with KF's annual flooding and salt flat proximities. 

Body: We must reach several milestones before completely presenting a conclusion to others. We must access the taxonomic identifications of all 
experimental 18S DNA from the KF sites. This will be compared to identifications of established data from B sites. The taxa of both sites must be attached
together after creating distinct and seperate phylogenetic trees. This will provide ease of access for future data manipulation. Inferences can be made
after: 1. extraneous background research 2. compilied phylogeny tree. 

Body: We expect accessible AMF phylogenetic data for benevolent and interested parties. The KF sites hold critically endangered legumes named Astragalus
applegatei, to whom share an obligate symbiosis. We wish to expand known knowledge about these legume by understanding their symbionts in context. If this
research provides more insight into species interactions, then people may better understand how to specifically expand applegatei's population count. I
opt to this in case others wish for its extant persistence.

#2nd commit a588e223d7a4022050f6c97ac836f4e7a8553d99
#2nd edit for v1 on 1125 PST 7 Nov 2022
#edit.id=js781

#LIST TO WRITE
data sources-review
bioinformatics tools - with citations

Header: Data Sources from Banyuasin and Klamath Falls Sites
Body: B sites data were pulled from Sefrila et al. 2021 article called "Diversity and abundanceof arbuscular fungi mycorrhizal (AMF) in rhizosphere Zea
mays in tidal swamp." Genus and species names were searched within the MaarjAM database from Öpik et al. 2010. The resulting 18S sequence data was curated
into fasta files for R. 

Body: KF sites data were sourced from the Klamath Falls fields. Raw 18S sequence datas were collected back from Argonne National Laboratory in Illinois.
The sequences were trimmed and processed for least amount of sequence artifacts. These sequences were searched within the MaarjAM database as well. The
two file libraries were curated into an Microsoft Excel spreadsheet. The taxon ranks in the spreadsheet were used to inform grouping within Python code, 
with the help of the ETE toolkit. 

#LIST TO WRITE
bioinformatics tools - with citations-review

#3rd commit 2fc0b59cfb78bceae25eccd6abe3fa146fb56d2f

#3rd edit for v2 on 2325 PST 7 Nov 2022
#edit.id=js781

Header: Klamath Falls site inspiration:
Body: Barroetavena, Carolina, et al. “Mycorrhizal Status of the Endangered Species Astragalus Applegatei Peck as Determined from a Soil Bioassay.”
Mycorrhiza, vol. 8, no. 2, Sept. 1998, pp. 117–19. DOI.org (Crossref), https://doi.org/10.1007/s005720050222.

Header: Environment for Tree Exploration (ETE) toolkit:
Body: Jaime Huerta-Cepas, Francois Serra and Peer Bork. ETE 3: Reconstruction, analysis and visualization of phylogenomic data. Mol Biol Evol 2016; doi:
10.1093/molbev/msw046

Header: Biopython library:
Body: Peter J. A. Cock, Tiago Antao, Jeffrey T. Chang, Brad A. Chapman, Cymon J. Cox, Andrew Dalke, Iddo Friedberg, Thomas Hamelryck, Frank Kauff, Bartek
Wilczynski, Michiel J. L. de Hoon: “Biopython: freely available Python tools for computational molecular biology and bioinformatics”. Bioinformatics 25
(11), 1422–1423 (2009). https://doi.org/10.1093/bioinformatics/btp163

Header: Indonesian site inspiration:
Body: Sefrila, Marlin, Munif Ghulamahdi, and Purwono Purwono. “Diversity and Abundance of Arbuscular Fungi Mycorrhizal (AMF) in Rhizosphere Zea Mays in
Tidal Swamp.” Biodiversitas Journal of Biological Diversity 22, no. 11 (November 1, 2021). https://doi.org/10.13057/biodiv/d221144.



